### ISSUE TITLE
[vote] Add blocks left in vote period indicator

### ISSUE THREAD

**Issue by marcopeereboom** [decred/politeiagui#668]
The GUI provides no feedback as to how many blocks/duration there is left on voting. We need to add some sort of counter as to ho many blocks there are left in the voting period.

**Comment by fernandoabolafio** [comment]
We currently have a voting duration indicator but its behavior isn't consistent. 
So here is the must fix list for this issue:
- Make sure proposals which received votes AND are active voting are still showing the voting expiration time.
- Make sure the remaining time of voting is accurate on the conversion from blocks left to time left.
- Make sure proposals on finished voting don't display the time left.

**Comment by thi4go** [comment]
Will be working on this!

**Comment by tiagoalvesdulce** [comment]
@thi4go the problem with the voting period indicator was solved but we are still not showing how many blocks are left. I suggest we do something like: `X blocks (estimated in 5min) left`

**Comment by thi4go** [comment]
Right! I'm redesigning that div, and will put the blocks like that @tiagoalvesdulce
